lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js

```diff
@@ -21,6 +21,21 @@ const Gatherer = require('../gatherer');
 /* global document,window */
 
```

Comment:
can you add a big @fileoverview comment block at the top that explains how this gatherer works?